packages/ember/src/index.ts

```diff
@@ -0,0 +1,43 @@
+export function setupAppsignal(appsignal: any, Ember = (window as any).Ember) {
+  const prevHandler = Ember.onerror
+
+  if (!appsignal) {
+    throw new Error("No instance of Appsignal supplied.")
```

Comment:
what's the advantage here of throwing an error over logging an error and returning early from the function?

Reply:
Fixed in 8ec5cef.